CHANGES.rst

```diff
@@ -2,11 +2,18 @@
  CHANGES
 =========
 
-4.0.2 (unreleased)
+4.1.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- The help namespace no longer modifies the global help object on
+  traversal. Instead it returns a new proxy object. This makes it
+  thread-safe. See `issue 4
+  <https://github.com/zopefoundation/zope.app.onlinehelp/issues/4>`_.
 
+- ``getTopicFor`` now really returns the first found topic in the
+  event that the object implements multiple interfaces that have
+  registered topics for the given view. Previously it would return the
+  topic for the least-specific interface.
```

Comment:
(Ah, I see you attached a review comment explaining this, but the code changed or something and it wasn't visible in the diff view.)